pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/SegmentStatusChecker.java

```diff
@@ -156,6 +157,8 @@ private void stopThread() {
   public void runSegmentMetrics() {
     if (!_pinotHelixResourceManager.isLeader()) {
       LOGGER.info("Skipping Segment Status check, not leader!");
+      _controllerMetrics.addMeteredGlobalValue(ControllerMeter.CONTROLLER_NOT_LEADER, 1L);
```

Comment:
No, we don't need to do this for retention manager, segment status checker, we only need it when we return NOT_LEADER as a part of LLC consumption. thanks,